lib/const.rb

```diff
@@ -51,8 +51,8 @@
 CREW_CACHE_ENABLED = ENV['CREW_CACHE_ENABLED']
 
 CREW_CONFLICTS_ONLY_ADVISORY = ENV['CREW_CONFLICTS_ONLY_ADVISORY']
-
-CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY = ENV['CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY']
+CREW_FHS_NONCOMPLIANCE_ERROR = ENV['CREW_FHS_NONCOMPLIANCE_ERROR']
```

Comment:
If you are going to replace `CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY`, you will also need to change all the packages that reference this constant.  Suggestion: Don't change. :)

Reply:
Changed back